package/MDAnalysis/analysis/base.py

```diff
@@ -450,58 +461,3 @@ def __init__(self, trajectory=None, *args, **kwargs):
                                                *args, **kwargs)
 
     return WrapperClass
-
-
-def _filter_baseanalysis_kwargs(function, kwargs):
```

Comment:
Due to the removal of the explicit verbose option, the tests of the _private_ method `_filter_baseanalysis_kwargs` failed. `_filter_baseanalysis_kwargs` splits and tests the keyword arguments of a function and the Base class. With the removal the `verbose` (the only explicit keyword argument) `_filter_baseanalysis_kwargs` will fail because the `inspect.getargspec ` will return `None`. I see two options for the future of `_filter_baseanalysis_kwargs `

1. **Keeping** `_filter_baseanalysis_kwargs`: Test for `None` and return an empty dictionary and the unchannged `kwargs`.

I would go for the second option:

2. **Remove** `_filter_baseanalysis_kwargs`: With the removal of the only keyword argument of `AnalysisBase` `_filter_baseanalysis_kwargs` lost its purpose. There is no possible clash of the keyword arguments between a passed function and `AnalysisBase` anymore. Additionally, option 1 one results in a lot of dead code which will only be executed if  another explicit keyword argument of `AnalysisBase` will be introduce in the future.

Reply:
Getting rid of it would be good.

Add a note to AnalysisBase that the call signature `trajectory, **kwargs` is fixed and part of the API specification.